app/templates/layout.html

```diff
@@ -77,20 +77,7 @@
       {% endif%}
     {% endwith %}
 
-
-    <!-- The page's main content should go on the left (66%) side -->
-    <div class="leftcontent">
-      {% block content %}{% endblock %}
-    </div>
-
-    <!-- Each page can have a window on the right side for different options (e.g. important info) -->
-    <!-- Jasmine: I tried to wrap this in the rightcontent tag, but it adds an extra shape; 
-          for now, the .html file it loads has its content wrapped in the rightcontent tag-->
-    <!-- Andy: The rightblock should be wrapped in the rightcontent tag now. 
-      I think the extra shape was caused by the extends statement in the important_info.html rendering duplicate html -->
-  <div class="rightcontent">
-      {% block rightblock %}{% endblock %}
-  </div>
+    <div class="maincontent">{% block content %}{% endblock %}</div>
```

Comment:
Now, `layout.html` simply has a placeholder for the main content block.

Reply:
yea, the main layout should be simple.

>`
<html>
    <head>
        stylesheets
        metatags
        <title></title>
    </head>
    <nav class=our_nav_class>
        { % block nav % }
            #with statement
        { % endblock % }
    </nav>
    <body>
        <main class=our_main_div_class>
            { % block body % }{ % endblock % }
        </main>
    </body>
    <footer>
        <div class=our_footer_div_class>
            { % block footer % }{ % endblock % }
        </div>
    </footer>
</html>`